recipe/meta.yaml

```
@@ -11,22 +11,23 @@ source:
     - 5eff1ecc8413b0dc93a1ab047d7fed751e6cb40e.patch  # [not win]
 
 build:
-  number: 0
+  number: 1
 
 requirements:
   build:
     - {{ compiler('cxx') }}
 
   host:
-    - boost-cpp {{ version }}.*
+    - libboost-devel {{ version }}.*
     - {{ mpi }}
 
   run:
-    - boost-cpp {{ version }}.*
+    # explicitly depend on -devel version because we need the headers
+    - libboost-devel {{ version }}.*
     - {{ mpi }}
 
   run_constrained:
-    - libboost <0
+    - libboost >=1.82
```

We dont want to mix versions of Boost. Shouldn't this be
```
 - libboost =={{ version }}
```

---

That's already taken care of by the run-requirement. In a way this is redundant, but I had to touch it otherwise the new builds cannot be pulled in. Still, no harm in making it tighter...
```suggestion
    - libboost {{ version }}.*
```